src/synapse/azext_synapse/_help.py

```diff
@@ -279,96 +331,95 @@
+        az synapse spark session reset-timeout --livy-id 1 --workspace-name testsynapseworkspace --spark-pool-name testsparkpool
 """
 
-helps['synapse spark session-statement'] = """
+helps['synapse spark statement'] = """
 type: group
-short-summary: Manage Synapse Spark Session Statements.
+short-summary: Manage Synapse Spark statements.
 """
 
-helps['synapse spark session-statement create'] = """
+helps['synapse spark statement create'] = """
 type: command
-short-summary: Submit a spark statement to a spark session.
+short-summary: Submit a Spark statement.
 examples:
-  - name: Submit a spark statement to a spark session.
+  - name: Submit a Spark statement.
+    text: |-
+        az synapse spark statement create  --session-id 1 --workspace-name testsynapseworkspace \\
+        --spark-pool-name testsparkpool --code "print('hello, Azure CLI')" --language pyspark
+  - name: Submit a Spark statement by reading code content from file.
     text: |-
-        az synapse spark session-statement create  --session-id 1 --workspace-name testsynapseworkspace \\
-        --spark-pool-name testsparkpool --code "print('hello, Azure CLI')" --kind pyspark
+        az synapse spark statement create  --session-id 1 --workspace-name testsynapseworkspace \\
+        --spark-pool-name testsparkpool --code @file-path --language pyspark
 """
 
-helps['synapse spark session-statement show'] = """
+helps['synapse spark statement show'] = """
 type: command
-short-summary: Get a spark statement with statement id.
+short-summary: Get a Spark statement.
 examples:
-  - name: Get a spark statement with statement id.
+  - name: Get a Spark statement.
     text: |-
-        az synapse spark session-statement show --id 1 --session-id 11 --workspace-name testsynapseworkspace \\
+        az synapse spark statement show --livy-id 1 --session-id 11 --workspace-name testsynapseworkspace \\
         --spark-pool-name testsparkpool
 """
 
-helps['synapse spark session-statement list'] = """
+helps['synapse spark statement list'] = """
 type: command
-short-summary: List all spark statements under the specify spark session.
+short-summary: List all Spark statements
 examples:
-  - name: List all spark statements under the specify spark session.
+  - name: List all Spark statements.
     text: |-
-        az synapse spark session-statement list --session-id 11 --workspace-name testsynapseworkspace \\
+        az synapse spark statement list --session-id 11 --workspace-name testsynapseworkspace \\
         --spark-pool-name testsparkpool
 """
 
-helps['synapse spark session-statement cancel'] = """
+helps['synapse spark statement cancel'] = """
 type: command
-short-summary: Cancel a spark statement with statement id.
+short-summary: Cancel a Spark statement.
 examples:
-  - name: Cancel a spark statement with statement id.
+  - name: Cancel a SPark statement.

```

Comment:
>SPark [](start = 19, length = 5)

Spark

Reply:
fixed

---
In reply to: [418047639](https://github.com/Azure/azure-cli-extensions/pull/1581#discussion_r418047639) [](ancestors = 418047639)